ui/pages/v2/Automations.tsx

```diff
@@ -10,10 +10,9 @@ type Props = {
 
 function Automations({ className }: Props) {
   const { data: automations, error, isLoading } = useListAutomations();
```

Comment:
```suggestion
  const { data: automations, error: requestError, isLoading } = useListAutomations();
```

To differentiate the diff types of errors we're juggling now too?

Reply:
resolved the type here
 https://github.com/weaveworks/weave-gitops/blob/c6436dc5ec747b1fec208a6e7937b0341bc1bf45/ui/components/Page.tsx#L19